doc/man3/RAND_bytes.pod

```diff
@@ -29,6 +33,15 @@ instance so that a compromise of the "public" PRNG instance will not
 affect the secrecy of these private values, as described in L<RAND(7)>
 and L<RAND_DRBG(7)>.
 
+RAND_bytes_ex() and RAND_priv_bytes_ex() are the same as RAND_bytes() and
+RAND_priv_bytes() except that they both take an additional I<ctx> parameter.
+The DRBG used for the operation is the public or private DRBG associated with
+the specified I<ctx>. The parameter can be NULL, in which case
```

Comment:
Does fips need to be mentioned here -  as a NULL context isnt so great for that case :)

Reply:
As far as end user applications are concerned it is perfectly ok to use a NULL context with the FIPS module. A NULL context gets translated into a concrete instance by libcrypto - so by the time it gets to the FIPS module one is always present. Our own internal code will have to be careful to always supply one if we're making these calls from inside the FIPS module. But these man pages are definitely targeting end users - not us.

Reply:
I assume you dont mean via openssl_ctx_get_concrete(NULL)

Reply:
Yes - I do (don't forget this happens in libcrypto not the FIPS module - so that's perfectly ok).